src/test/kotlin/lotto/domain/LottoStatisticsTest.kt

```diff
@@ -10,37 +10,38 @@ import org.junit.jupiter.params.provider.CsvSource
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class LottoStatisticsTest {
 
-    val winningLotto = WinningLotto(Lotto(setOf(1, 2, 3, 4, 5, 6)))
+    val winningLotto = WinningLotto(Lotto(listOf(1, 2, 3, 4, 5, 6).map { LottoNumber(it) }.toSet()), LottoNumber(45))
```

Comment:
```suggestion
   private val winningLotto = WinningLotto(
        lotto(1,2,3,4,5,6),
        LottoNumber(45)
    )
    private fun lotto(vararg numbers: Int): Lotto {
        return Lotto(numbers.map { LottoNumber(it) }.toSet())
    }
```
lotto를 생성해 줄 함수를 하나 만들면 테스트 코드에 대한 가독성이 좋아지지 않을까요~?